.changeset/calm-cobras-tease.md

```diff
@@ -0,0 +1,5 @@
+---
+"setup-github-token": minor
```

Comment:
I think this would be considered a `patch`

Reply:
All set.